Languages/French/Keyed/Keyed-French.xml

```diff
@@ -25,9 +26,12 @@
     <Fluffy.ModManager.DependencyUnknownVersion>Dépend de {0}, mais sa version est inconnue.</Fluffy.ModManager.DependencyUnknownVersion>
     <Fluffy.ModManager.DependencyNotFound>Dépend de {0}, qui n'est pas chargé.</Fluffy.ModManager.DependencyNotFound>
     <Fluffy.ModManager.DependencyWrongVersion>Dépend de {0}, v{1} est chargé.</Fluffy.ModManager.DependencyWrongVersion>
```

Comment:
English text here is `<Fluffy.ModManager.DependencyWrongVersion>Depends on {0} version {1}, version {2} is loaded.</Fluffy.ModManager.DependencyWrongVersion>`